src/main/resources/templates/login.html

```diff
@@ -14,8 +14,11 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-heading-examp
                         <p>
 
                             <span th:if="${param.error[0] == 'failed'}">You have entered the wrong email address or password, please try again.</span>
-                            <span th:if="${param.error[0] == 'locked'}">Your account has been locked. Please contact us to <a
-                                    href="mailto:feedback@cslearning.gov.uk">unlock your account.</a></span>
+                            <span th:if="${param.error[0] == 'locked'}"><a href="/reset" class="push-bottom">Reset your password to unlock your account</a></span>
+
+
+<!--                            <span th:if="${param.error[0] == 'locked'}">Your account has been locked. Please contact us to -->
+<!--                                <a href="mailto:feedback@cslearning.gov.uk">unlock your account.</a></span>-->
```

Comment:
Remove comment

Reply:
Is this still needed?

Reply:
no, was for reference only; can be removed.